docs/releases/updating_ez_platform.md

```diff
@@ -120,6 +120,10 @@ On PHP conflict | 16.02 and later requires PHP 5.5 or higher
 
 Because from release 16.02 onwards eZ Platform is compatible only with PHP 5.5 and higher, the update command above will fail if you use an older PHP version. Please update PHP to proceed.
 
+!!! Common errors
```

Comment:
For the note box to generate correctly, you need to add a note type (note/tip/caution) and put the title in quotation marks:
`!!! note "Common errors"`